R/modelling.R

```diff
@@ -514,21 +515,19 @@ extract_seromodel_summary <- function(seromodel_object,
 #' @export
 get_prev_expanded <- function(foi,
                               serodata,
-                              predicted_prev_lower_quantile = 0.1,
-                              predicted_prev_upper_quantile = 0.9,
-                              bin_data = FALSE) {
-  dim_foi <- dim(foi)[2]
-  # TODO: check whether this conditional is necessary
-  if (dim_foi < 80) {
-    oldest_year <- 80 - dim_foi + 1
-    foin <- matrix(NA, nrow = dim(foi)[1], 80)
-    foin[, oldest_year:80] <- foi
-    foin[, 1:(oldest_year - 1)] <- rowMeans(foi[, 1:5])
-  } else {
-    foin <- foi
+                              alpha = 0.05,
+                              bin_data = FALSE,
```

Comment:
Right now this function is used only to flexibilize the visualisation functions `plot_seroprev` and `plot_seroprev_fitted`, so I think it would be better to make it a internal function rather than asking the user to call `prepare_bin_data`.